packages/web-app-files/src/fileTypeColorMappings.json

```diff
@@ -0,0 +1,107 @@
+{
+    "7z": "#FFCF01",
+    "accdb": "var(--oc-color-text-default",
+    "ai": "#FF6F00",
+    "apk": "#FFCF01",
+    "bash": "var(--oc-color-text-default",
+    "bz2": "#FFCF01",
+    "c++": "var(--oc-color-text-default",
+    "c": "var(--oc-color-text-default",
+    "cb7": "var(--oc-color-text-default",
```

Comment:
@lookacat this is where it takes the light/dark colors ;)

Reply:
@pascalwengerter moved it to ODS, its now using only icon color tokens for themes :)

Reply:
Thx!